[2024-10-25T15:41:10.143+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:10.146+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/wiki_view.py for tasks to queue
[2024-10-25T15:41:10.153+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:41:10.152+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:10.315+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:41:10.304+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/wiki_view.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/wiki_view.py", line 8, in <module>
    from dags.function_csv.database_function import create_postgres_table, load_csv_to_database
ModuleNotFoundError: No module named 'dags'
[2024-10-25T15:41:10.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:10.375+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/wiki_view.py took 0.255 seconds
[2024-10-25T15:41:40.628+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:40.630+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/wiki_view.py for tasks to queue
[2024-10-25T15:41:40.636+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:41:40.635+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:40.727+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:41:40.719+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/wiki_view.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/wiki_view.py", line 8, in <module>
    from dags.function_csv.database_function import create_postgres_table, load_csv_to_database
ModuleNotFoundError: No module named 'dags'
[2024-10-25T15:41:40.731+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:41:40.788+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/wiki_view.py took 0.177 seconds
[2024-10-25T15:42:10.995+0000] {processor.py:186} INFO - Started process (PID=76) to work on /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:10.997+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/wiki_view.py for tasks to queue
[2024-10-25T15:42:11.004+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:42:11.002+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:11.104+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:42:11.094+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/wiki_view.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/wiki_view.py", line 8, in <module>
    from dags.function_csv.database_function import create_postgres_table, load_csv_to_database
ModuleNotFoundError: No module named 'dags'
[2024-10-25T15:42:11.108+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:11.160+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/wiki_view.py took 0.192 seconds
[2024-10-25T15:42:41.353+0000] {processor.py:186} INFO - Started process (PID=84) to work on /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:41.355+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/wiki_view.py for tasks to queue
[2024-10-25T15:42:41.362+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:42:41.361+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:41.462+0000] {logging_mixin.py:190} INFO - [2024-10-25T15:42:41.456+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/wiki_view.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/wiki_view.py", line 8, in <module>
    from dags.function_csv.database_function import create_postgres_table, load_csv_to_database
ModuleNotFoundError: No module named 'dags'
[2024-10-25T15:42:41.466+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/wiki_view.py
[2024-10-25T15:42:41.500+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/wiki_view.py took 0.167 seconds
